_posts/2020-10-05-Evidence_locker_overview.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+category: Architecture
+title: Evidence locker overview
+---
+
+At the core of the [Auditree framework][] is a [Git][] repository known as the
+"evidence locker".  Each Auditree execution environment requires a dedicated
+evidence locker in order to execute.  The Auditree framework can either clone
+an evidence locker from a remote Git repository hosting service (GitHub,
+GitLab, ...) or create a new local locker for you.  Serving as the Auditree
+data store, the evidence locker houses evidence gathered by fetchers, evidence
+generated by checks and/or evidence planted by the [Auditree Plant][] tool.
+
+## Local vs. remote
+
+The reasoning behind our choice of Git as the Auditree data store is
+outlined nicely in our [previous post][] but in a nutshell, an evidence locker
+uses Git as an efficient, reliable, tamper evident data store.  Although it is
+possible to execute Auditree with only a local evidence locker, a typical
+implementation of an Auditree execution environment will pull down a local
+clone of a remote evidence locker, perform Auditree operations, store the
+results of those operations as evidence in the cloned local locker and once
+complete, push changes back to the remote evidence locker.
+
+## Organization
+
+Evidence locker content is namespaced into four top level folders within the
+locker and manages that content on an ongoing basis.  The folders are:
+
+- **raw**: All evidence retrieved by fetchers is stored here.
+- **reports**: All report evidence generated by checks is stored here.
+- **external**: All evidence gathered or generated outside of the Auditree
+framework and planted by [Auditree Plant][] is stored here.
+- **notifications**: All check notifications sent to the locker are stored
+here.
+
+## OK, but what makes it a locker?
+
+...in a word, **metadata**.
+
+As noted earlier, all files are organized according to their data type but
+the Auditree framework still needs a way to treat these files as evidence and
+that's where the metadata comes in.  The top level folders are purposefully
+shallow. Each folder that contains evidence (`notifications` are a special
+case, and not considered evidence) can be further organized with sub-folders
+per category of evidence under `raw`, `reports` and `external`.  Each evidence
```

Comment:
"...category of evidence under `raw`, `reports` and `external`, for example by tool or service provider."